src/NServiceBus.AcceptanceTests/Core/Installers/When_installer_behavior_not_specified.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace NServiceBus.AcceptanceTests.Installers
+{
+    using System.Diagnostics;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using Configuration.AdvanceExtensibility;
+    using EndpointTemplates;
+    using Installation;
+    using NUnit.Framework;
+
+    public class When_installer_behavior_not_specified : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_not_run_installers_when_no_debugger_attached()
+        {
+            if (Debugger.IsAttached)
+            {
+                Assert.Ignore("This test must run without a debugger attached but a debugger is attached.");
+            }
+
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<EndpointWithInstaller>()
+                .Done(c => c.EndpointsStarted)
+                .Run();
+
+            Assert.IsFalse(context.InstallerCalled);
+        }
+
+        [Test]
+        public async Task Should_run_installers_when_debugger_attached()
+        {
+            if (!Debugger.IsAttached)
```

Comment:
that's a bit shitty, but maybe this is better than not having this test? I'm on the fence.